google/cloud/translate/v3beta1/translate_gapic.yaml

```diff
@@ -110,126 +99,20 @@ interfaces:
-    required_fields:
-    - contents
-    - target_language_code
-    retry_codes_name: non_idempotent
     retry_params_name: default
-    field_name_patterns:
-      parent: location
-    timeout_millis: 60000
   - name: DetectLanguage
-    flattening:
-      groups:
-      - parameters:
-        - parent
-        - model
-        - mime_type
-    required_fields:
-    retry_codes_name: non_idempotent
     retry_params_name: default
-    field_name_patterns:
-      parent: location
-    timeout_millis: 60000
   - name: GetSupportedLanguages
-    flattening:
-      groups:
-      - parameters:
-        - parent
-        - display_language_code
-        - model
-    required_fields:
-    retry_codes_name: idempotent
     retry_params_name: default
-    field_name_patterns:
-      parent: location
-    timeout_millis: 60000
   - name: BatchTranslateText
-    required_fields:
-    - source_language_code
-    - target_language_codes
-    - input_configs
-    - output_config
-    retry_codes_name: non_idempotent
     retry_params_name: default
-    field_name_patterns:
-      parent: location
-    long_running:
-      return_type: google.cloud.translation.v3beta1.BatchTranslateResponse
-      metadata_type: google.cloud.translation.v3beta1.BatchTranslateMetadata
-      initial_poll_delay_millis: 500
-      poll_delay_multiplier: 1.5
-      max_poll_delay_millis: 5000
-      total_poll_timeout_millis: 300000
-    timeout_millis: 60000
   - name: CreateGlossary
-    flattening:
-      groups:
-      - parameters:
-        - parent
-        - glossary
-    required_fields:
-    - parent
-    - glossary
-    retry_codes_name: non_idempotent
     retry_params_name: default
-    field_name_patterns:
-      parent: location
-    long_running:
-      return_type: google.cloud.translation.v3beta1.Glossary
-      metadata_type: google.cloud.translation.v3beta1.CreateGlossaryMetadata
-      initial_poll_delay_millis: 500
-      poll_delay_multiplier: 1.5
-      max_poll_delay_millis: 5000
-      total_poll_timeout_millis: 300000
-    timeout_millis: 60000
   - name: ListGlossaries
-    flattening:
-      groups:
-      - parameters:
-        - parent
-        - filter
-    required_fields:
-    page_streaming:
-      request:
-        page_size_field: page_size
-        token_field: page_token
-      response:
-        token_field: next_page_token
-        resources_field: glossaries
-    retry_codes_name: idempotent
     retry_params_name: default
-    field_name_patterns:
-      parent: location
-    timeout_millis: 60000
   - name: GetGlossary
-    flattening:
-      groups:
-      - parameters:
-        - name
-    required_fields:
-    - name
-    retry_codes_name: idempotent
     retry_params_name: default
-    field_name_patterns:
-      name: glossary
-    timeout_millis: 60000
   - name: DeleteGlossary
-    flattening:
-      groups:
-      - parameters:
-        - name
-    required_fields:
-    - name
     retry_codes_name: idempotent
```

Comment:
The default here would have been non_idempotent. The API might be wrong on this one, because it is server-side idempotent but probably not client-side idempotent.